doc/api/child_process.md

```diff
@@ -147,7 +147,9 @@ added: v0.1.90
 * Returns: {ChildProcess}
 
 Spawns a shell then executes the `command` within that shell, buffering any
-generated output.
+generated output. When specifying paths, spaces need to be dealt with 
```

Comment:
looks good, but mention that the shell specials depend on the shell, and maybe link to where it docs what shell is used on what system?